ci-goreleaser/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM golang:1.10.1
 
 LABEL maintainer="bjorn.erik.pedersen@gmail.com"
 
-ENV GORELEASER_VERSION=0.34.2
+ENV GORELEASER_VERSION=0.79.0
 ENV GORELEASER_SHA=013b6289cff41b8afca638007a727170efc8d61469405e2a5d30edf1343e5b68
```

Comment:
I assume the new release has a different SHA hash.

Reply:
whoops